lib/common_test/src/ct_slave.erl

```diff
@@ -309,7 +309,14 @@ is_started(ENode) ->
 
 % make a Erlang node name from name and hostname
 enodename(Host, Node) ->
-    list_to_atom(atom_to_list(Node)++"@"++atom_to_list(Host)).
+    case lists:splitwith(fun($@) -> false;
+                            (_)  -> true
+                         end, atom_to_list(Node)) of
+        {Head, [$@,_|_]=HostPart} ->
+            list_to_atom(Head ++ HostPart);
```

Comment:
Seems that Node would work just as well. That also means that you can use `lists:member/2` instead of `lists:splitwith/2`.